bzt/engine.py

```diff
@@ -1036,13 +1036,18 @@ def get_requests(self):
             timeout = req.get("timeout", None)
             think_time = req.get("think-time", None)
 
-            body = None
-            bodyfile = req.get("body-file", None)
-            bodyfile_path = self.engine.find_file(bodyfile)
-            if bodyfile_path:
+            body = req.get('body', None)
+            body_file = req.get('body-file', None)
+            if body and body_file:
+                body_file = None
+                req['body-file'] = None
+
+            if body_file:  # insert file into body
+                bodyfile_path = self.engine.find_file(body_file)
                 with open(bodyfile_path) as fhd:
-                    body = fhd.read()
-            body = req.get("body", body)
+                    req['body'] = fhd.read()
+                    body = req['body']
+                    req['body-file'] = None
```

Comment:
I think it's wrong. Can we keep body-file in config always? Just yield the body.